docs/root/api-v3/config/filter/generic/generic.rst

```diff
@@ -1,4 +1,4 @@
-Dubbo filters
+Generic filters
 ==============
```

Comment:
you will need to make the unerline match the title length

Reply:
done.